test/Libraries/RevitNodesTests/RevitNodesTests.csproj

```diff
@@ -39,21 +39,21 @@
     <Prefer32Bit>false</Prefer32Bit>
   </PropertyGroup>
   <ItemGroup>
-    <Reference Include="DSCoreNodes, Version=1.0.1.1523, Culture=neutral, processorArchitecture=MSIL">
+    <Reference Include="DSCoreNodes">
```

Comment:
Let's add some `<HintPath>` here for consistency and to ensure that the assembly can be resolved :)